src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -511,6 +520,14 @@ export class ResultsViewPageStore {
             },
             { fireImmediately: true }
         );
+
+        // this localStorage variable will be used to get relevant hgvsg url
+        if (this.grch38GenomeNexusUrl) {
+            getBrowserWindow().localStorage.setItem(
```

Comment:
i forget, why are we puting this in localstorage?

Reply:
@alisman this comes from mutation mapper tool.
In mutation mapper tool, when people choose "GRCh37" or "GRCh38", we need to provide separate genome nexus instances. And in order to update genome nexus clients, we need to reload the page when people change their selection, so we need this variable to store their selection.
In this pr, I use this variable to create [hgvsg links](https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/api/urls.ts#L219-L224)

Reply:
but this seems like something that belongs in the mutation mapper page.  It's particular to that page, not to the this store, right?